### PR TITLE
Capstone Project

### PR DESCRIPTION
**Pull request by Boku52500**
- Added linters.yml
- Created Mobile pages for both main and about.
- Created a dynamic Speakers section with javaScript array.
- Adjusted all the contect to each resolution.
- Created nav bar for 768px and wider.
- Created functional hamburger menu dropdown for mobile version.
- Created a README file.
- Fixed linters.
- Recorded an explanation video for this project which you can check out with this [link.](https://www.veed.io/view/097cd8d8-5173-4e1f-931a-11081597ab13?sharingWidget=true)

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+              <a href="index.html#program">Program</a>
+              <a href="#">Join</a>
+              <a href="#sponsors">Sponsor</a>
+              <a href="#">News</a>
+            </div>
+            <div class="campaign-container">
+              <button class="campaign-button">Chess Summit</button>
+            </div>
+          </div>
+        </nav>
```
Comment (ridwanediallo) [inline about.html:48]: - [ ] Hey @Boku52500 you did a great job on this project. 👏 However, let's make it look more like the design template. I would recommend adding the `nav bar ` with the social media icons at the top, like in the template.

### The template design 👇️
Screenshot from 2022-10-21 01-00-48
### Your design 👇️
Screenshot from 2022-10-21 01-07-17

File: script.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+const speakers = [{
+  photo: 'images/portrait1.png',
```
Comment (ridwanediallo) [inline script.js:2]: - [ ] Please make sure to resize the images in the speaker section, they are too big compared to those on the template.
### Your images 👇️
Screenshot from 2022-10-21 01-10-53
### On the template  👇️
Screenshot from 2022-10-21 01-15-06

File: style.css [hunk 3]
```diff
@@ -0,0 +1,650 @@
...
+.about {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  background-repeat: no-repeat;
+  padding: 20px 5%;
+  background-position-y: 17%;
+  background-origin: content-box;
+  background-color: #f7f7f7;
+  margin-top: 5%;
```
Comment (ridwanediallo) [inline style.css]: - [ ] Please remove this `margin-top: 5%;` from your about section, it is adding a blank space at top of your page for the mobile view.

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @Boku52500 , 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+          <h1>Partner</h1>
+          <div class="line-small"></div>
+          <div class="sponsor-names">
+            <img src="images\sponsor1.png" alt="">
+            <img src="images\sponsor2.png" alt="">
+            <img src="images\sponsor3.png" alt="">
+            <img src="images\sponsor4.png" alt="">
+            <img src="images\sponsor5.png" alt="">
+          </div>
+        </div>
```
Comment (Santiago220991) [inline about.html:109]: [OPTIONAL] Kindly make sure that your project sticks as much as possible to the template provided. 
- This section is not included in the template provided. :eyes: :eyes:  You could remove it.

Your design
image

Template design
image

File: about.html [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+        <footer class="about-foot">
+          <div class="chess-footer-img">
+            <img src="images\chess-logo.png" alt="" />
+          </div>
+          <div class="footer-text">
+            2022 Chess Global Summit. Some Rights Reserved.
+          </div>
+        </footer>
+    
+      </section>
```
Comment (Santiago220991) [inline about.html:120]: [OPTIONAL] Kindly ensure that your project sticks as much as possible to the template provided. The color of this section in the desktop version is different from the original design. Kindly change this. :pray: :pray: Also pay attention please to the font color.

Your design
image

Template design
image

**Review by Santiago220991 — APPROVED** [review]
Hi @Boku52500, 
I'm Santiago your reviewer for this PR. :wave: :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
:heavy_check_mark: Applied changes suggested by the last reviewer.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Followed GitHub flow.

## Optional suggestions
[OPTIONAL] I think is a good idea to add some animations to make your project more dynamic. :+1: :smiley: :smiley: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**